src/sagas.js

```diff
@@ -138,6 +139,7 @@ function* fetchAllVms (action) {
     yield put(removeMissingVms({ vmIdsToPreserve }))
 
     yield put(updateVms({ vms: internalVms, copySubResources: true }))
+    yield fetchVmsSessions({ vms: internalVms })
```

Comment:
Right, it's to costly since it's a per-VM call.

I agree with @jelkosz , let's try usability when triggered within console retrieval.